### PR TITLE
Toggle js init value

### PR DESCRIPTION
**Pull request by seriewe**
Init toggle value as selected if is checked attribute set

### CODE REVIEW

**Comment by Jahnp** [comment]
Thanks for the PR @seriewe. What's the purpose of these changes? What are you trying to solve?

**Comment by seriewe** [comment]
Hi, if my toggle is checked on server side and the view is rendered, the checkbox was not selected by fabric ui. So I check the "checked" Attribute and set is-selected. Sorry for the confusing stuff with the border inside the pull request. The important stuff is inside the Toggle.ts ctor.

File: src/components/Toggle/Toggle.ts [hunk 1]
```diff
@@ -35,7 +40,6 @@ namespace fabric {
     }
 
     private _toggleHandler(): void {
```
Comment (GageAmes) [inline src/components/Toggle/Toggle.ts]: `_toggleHandler()` now does nothing yet the method is still used as a callback to a few events.  Was removing this method's body intentional?

File: src/components/Toggle/Toggle.ts [hunk 2]
```diff
@@ -35,7 +40,6 @@ namespace fabric {
...
-      this._toggleField.classList.toggle("is-selected");
```
Comment (GageAmes) [inline src/components/Toggle/Toggle.ts]: `_toggleHandler()` now does nothing yet the method is still used as a callback to a few events.  Was removing this method's body intentional?

Comment (seriewe) [inline src/components/Toggle/Toggle.ts]: Hi, no this was a mistake. The _toggleHandler() should still stay the same. Should I create a new PullRequest?

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).

**Pull request closed without merging**